### ISSUE TITLE
Failed to extract modpack

### ISSUE THREAD

**Issue by hdusahd** [MultiMC/Launcher#4182]
### Description of bug

Whenever I try to download better minecraft, it keeps saying failed to extract modpack. What do I do?

### Steps to reproduce

add instance, go to curseforge, select betterminecraft 1.16.5 v41 and try to download

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by Mr-melonU** [comment]
You can download and extract the modpack manualy then compress the content into a new zip file,then in MMC click Add Instance--Import from zip.

**Comment by Neutrovertido** [comment]
I can confirm that this issue also occurs with modded/vanilla instances exported to `.zip` files and attempting to import them (oddly enough the importing did work when on Linux, specifically Pop!_OS 22.04).

For Windows the solution proposed by @Mr-melonU proved to be a functioning workaround.

**Comment by BrandonMCMFG** [comment]
I can confirm that this issue also occures with foreign characters in filenames, I have a resourcepack that has russian characters and it failed to import with that, I removed that and now I can import the pack.

**Comment by flarefluffsune** [comment]
I'm sorry to necro this issue but I'm also encountering this issue now (And this issue is still open) while trying to install RLCraft 2.9.3, I don't know why its happening but I thought I should report it.

**Comment by MangyCat** [comment]
> I'm sorry to necro this issue but I'm also encountering this issue now (And this issue is still open) while trying to install RLCraft 2.9.3, I don't know why its happening but I thought I should report it.

I'm having that same problem too. Don't know why its unsolved. Very sorry for necro

**Comment by morgantic** [comment]
Same issue here. It would be really good to get a detailed error *why* it failed.

I upgraded my 1.21.4 instance to 1.21.5 and changed all the mods over. It worked in 1.21.4 and now doesn't work in 1.21.5. This renders one of the most useful features, exporting instances with a subset of all the folders and files, completely useless...

**EDIT** out of desperation, I just tried to manually extract the zip into the instances folder and it **worked**... So it can't really be an issue with the zip file...?!

**Comment by MangyCat** [comment]
This issue? It's still open? can someone close it at this point?

**Comment by morgantic** [comment]
Is this fixed?

**Comment by BuckinghamStudios** [comment]
> Is this fixed?

yeah i havent had an issue with this for like 4 years now, just never closed it lol, mb

**Comment by morgantic** [comment]
Uhm, did you read my comment from April?

Guess I'll go open my own bug report then ¯\\\_(ツ)\_/¯

**Comment by BuckinghamStudios** [comment]
> Uhm, did you read my comment from April?
> 
> Guess I'll go open my own bug report then ¯\_(ツ)_/¯

i reopened for u, i dont really play the game so i didnt know if it was fully fixed or not

**Comment by morgantic** [comment]
Thank you 😁

**Comment by MangyCat** [comment]
It took them so long to close issue i already made 20 projects on github :trollface: 

edit: its reopened 🫃
